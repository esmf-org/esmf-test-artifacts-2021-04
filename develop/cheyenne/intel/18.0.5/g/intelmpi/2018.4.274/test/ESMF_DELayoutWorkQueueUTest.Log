20210411 061800.987 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.987 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210411 061800.987 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210411 061800.987 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210411 061800.987 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210411 061800.987 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.987 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_04-7-g3098c96bb1
20210411 061800.987 INFO             PET0 ESMF library build date/time: "Apr 11 2021" "05:58:00"
20210411 061800.987 INFO             PET0 ESMF library build location : /glade/scratch/mpotts/intel_18.0.5_intelmpi_g
20210411 061800.987 INFO             PET0 ESMF_COMM                   : intelmpi
20210411 061800.987 INFO             PET0 ESMF_MOAB                   : enabled
20210411 061800.987 INFO             PET0 ESMF_LAPACK                 : enabled
20210411 061800.987 INFO             PET0 ESMF_NETCDF                 : enabled
20210411 061800.987 INFO             PET0 ESMF_PNETCDF                : disabled
20210411 061800.987 INFO             PET0 ESMF_PIO                    : enabled
20210411 061800.987 INFO             PET0 ESMF_YAMLCPP                : enabled
20210411 061800.988 INFO             PET0  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210411 061800.988 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210411 061800.989 INFO             PET0  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210411 061800.989 INFO             PET0  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210411 061800.989 INFO             PET0  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210411 061800.990 INFO             PET0  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210411 061807.016 INFO             PET0  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210411 061807.016 INFO             PET0  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210411 061807.016 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20210411 061807.016 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20210411 061807.017 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20210411 061807.017 DEBUG            PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20210411 061807.030 INFO             PET0  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210411 061809.586 INFO             PET0  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210411 061809.602 INFO             PET0  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210411 061809.603 INFO             PET0  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210411 061809.603 INFO             PET0 Finalizing ESMF
20210411 061800.988 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.988 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210411 061800.988 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210411 061800.988 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210411 061800.988 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210411 061800.988 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.988 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_04-7-g3098c96bb1
20210411 061800.988 INFO             PET1 ESMF library build date/time: "Apr 11 2021" "05:58:00"
20210411 061800.988 INFO             PET1 ESMF library build location : /glade/scratch/mpotts/intel_18.0.5_intelmpi_g
20210411 061800.988 INFO             PET1 ESMF_COMM                   : intelmpi
20210411 061800.988 INFO             PET1 ESMF_MOAB                   : enabled
20210411 061800.988 INFO             PET1 ESMF_LAPACK                 : enabled
20210411 061800.988 INFO             PET1 ESMF_NETCDF                 : enabled
20210411 061800.988 INFO             PET1 ESMF_PNETCDF                : disabled
20210411 061800.988 INFO             PET1 ESMF_PIO                    : enabled
20210411 061800.988 INFO             PET1 ESMF_YAMLCPP                : enabled
20210411 061800.988 INFO             PET1  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210411 061800.988 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210411 061800.989 INFO             PET1  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210411 061800.989 INFO             PET1  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210411 061805.015 INFO             PET1  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210411 061805.016 INFO             PET1  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210411 061807.016 INFO             PET1  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210411 061807.016 INFO             PET1  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210411 061807.016 DEBUG            PET1 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.016 DEBUG            PET1 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.030 INFO             PET1  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210411 061809.586 INFO             PET1  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210411 061809.586 INFO             PET1  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210411 061809.587 INFO             PET1  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210411 061809.587 INFO             PET1 Finalizing ESMF
20210411 061800.987 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.987 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210411 061800.987 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210411 061800.987 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210411 061800.987 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210411 061800.987 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.987 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_04-7-g3098c96bb1
20210411 061800.987 INFO             PET2 ESMF library build date/time: "Apr 11 2021" "05:58:00"
20210411 061800.987 INFO             PET2 ESMF library build location : /glade/scratch/mpotts/intel_18.0.5_intelmpi_g
20210411 061800.987 INFO             PET2 ESMF_COMM                   : intelmpi
20210411 061800.987 INFO             PET2 ESMF_MOAB                   : enabled
20210411 061800.987 INFO             PET2 ESMF_LAPACK                 : enabled
20210411 061800.987 INFO             PET2 ESMF_NETCDF                 : enabled
20210411 061800.987 INFO             PET2 ESMF_PNETCDF                : disabled
20210411 061800.987 INFO             PET2 ESMF_PIO                    : enabled
20210411 061800.987 INFO             PET2 ESMF_YAMLCPP                : enabled
20210411 061800.988 INFO             PET2  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210411 061800.988 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210411 061800.989 INFO             PET2  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210411 061800.989 INFO             PET2  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210411 061807.016 INFO             PET2  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210411 061807.016 INFO             PET2  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210411 061807.016 INFO             PET2  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210411 061807.016 INFO             PET2  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210411 061807.016 DEBUG            PET2 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.016 DEBUG            PET2 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.030 INFO             PET2  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210411 061809.550 INFO             PET2  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210411 061809.550 INFO             PET2  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210411 061809.551 INFO             PET2  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210411 061809.551 INFO             PET2 Finalizing ESMF
20210411 061800.988 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.988 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210411 061800.988 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210411 061800.988 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210411 061800.988 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210411 061800.988 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210411 061800.988 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_04-7-g3098c96bb1
20210411 061800.988 INFO             PET3 ESMF library build date/time: "Apr 11 2021" "05:58:00"
20210411 061800.988 INFO             PET3 ESMF library build location : /glade/scratch/mpotts/intel_18.0.5_intelmpi_g
20210411 061800.988 INFO             PET3 ESMF_COMM                   : intelmpi
20210411 061800.988 INFO             PET3 ESMF_MOAB                   : enabled
20210411 061800.988 INFO             PET3 ESMF_LAPACK                 : enabled
20210411 061800.988 INFO             PET3 ESMF_NETCDF                 : enabled
20210411 061800.988 INFO             PET3 ESMF_PNETCDF                : disabled
20210411 061800.988 INFO             PET3 ESMF_PIO                    : enabled
20210411 061800.988 INFO             PET3 ESMF_YAMLCPP                : enabled
20210411 061800.988 INFO             PET3  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20210411 061800.988 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210411 061800.989 INFO             PET3  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20210411 061800.989 INFO             PET3  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20210411 061800.989 INFO             PET3  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20210411 061800.990 INFO             PET3  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20210411 061807.016 INFO             PET3  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20210411 061807.016 INFO             PET3  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20210411 061807.016 DEBUG            PET3 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.016 DEBUG            PET3 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20210411 061807.017 INFO             PET3  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20210411 061809.546 INFO             PET3  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20210411 061809.547 INFO             PET3  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20210411 061809.547 INFO             PET3  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20210411 061809.547 INFO             PET3 Finalizing ESMF
