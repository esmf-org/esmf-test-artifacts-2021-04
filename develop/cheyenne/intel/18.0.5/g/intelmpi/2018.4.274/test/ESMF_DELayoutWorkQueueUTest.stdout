  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            0  time:   2.708435058593750E-004
 <round 1> PET            3  time:   2.770423889160156E-004
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            1  time:    4.02662205696106     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            2  time:    6.02675580978394     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            2  time:    2.50509095191956     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            3  time:    2.51993989944458     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            1  time:    2.56909012794495     
 <round 2> PET            0  time:    2.55597710609436     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PET           3  Test Elapsed Time    6107.10200000000       msec.
  PET           2  Test Elapsed Time    6124.86900000000       msec.
  PET           1  Test Elapsed Time    6058.75000000000       msec.
  PET           0  Test Elapsed Time    16057.0990000000       msec.
